src/Rap2hpoutre/LaravelLogViewer/LaravelLogViewer.php

```diff
@@ -18,6 +18,13 @@ class LaravelLogViewer
      */
     private $folder;
 
+    private static $patterns = [
+        'all_log_data' => '/\[\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}([\+-]\d{4})?\].*/',
+        'current_log_data_first' => '/^\[(\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}([\+-]\d{4})?)\](?:.*?(\w+)\.|.*?)',
+        'current_log_data_second' => ': (.*?)( in .*?:[0-9]+)?$/i',
```

Comment:
`current_log_data_first` and `current_log_data_first` is a bit bizarre. What about this: 

```php
'current_log_data' => [
    '/^\[(\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}([\+-]\d{4})?)\](?:.*?(\w+)\.|.*?)',
    ': (.*?)( in .*?:[0-9]+)?$/i',
]
```

What are your thoughts?